app/controllers/application_controller.rb

```diff
@@ -1,5 +1,10 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+
   protect_from_forgery with: :exception
-end
+  include SessionsHelper
+  include PostsHelper
+
+end 
+
+
+
```

Comment:
if you are using Sublime Text at this point, see if you can configure it to remove trailing whitespace from lines and trailing blank lines from files